sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -649,14 +652,46 @@ public String getDefaultLanguage() {
      *
      * @return A {@link Mono} contains the {@link DocumentSentiment analyzed document sentiment} of the document.
      *
-     * @throws NullPointerException if {@code document} is {@code null}.
+     * @throws NullPointerException if {@code document} is null.
      * @throws TextAnalyticsException if the response returned with an {@link TextAnalyticsError error}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<DocumentSentiment> analyzeSentiment(String document, String language) {
+        return analyzeSentiment(document, language, null);
+    }
+
+    /**
+     * Returns a sentiment prediction, as well as confidence scores for each sentiment label (Positive, Negative, and
+     * Neutral) for the document and each sentence within it. If the {@code includeOpinionMining} of
+     * {@link AnalyzeSentimentOptions} set to true, the output will include the opinion mining result. It mined the
+     * opinions of a sentence and conduct more granular analysis around the aspects of a product or service
+     * (also known as aspect-based sentiment analysis).
+     *
+     * <p><strong>Code Sample</strong></p>
+     * <p>Analyze the sentiments in a document with a provided language representation and
+     * {@link AnalyzeSentimentOptions} options. Subscribes to the call asynchronously and prints out the
```

Comment:
Should this example show opinion mining aspect of analyzing sentiment?

Reply:
Yup. Refactored the samples.